plugins/inputs/vsphere/README.MD

```diff
@@ -0,0 +1,350 @@
+# VMware vSphere Input Plugin
+
+The VMware vSphere plugin uses the vSphere API to gather metrics from multiple vCenter servers.
+ 
+* Clusters
+* Hosts
+* VMs
+* Data stores
+
+## Configuration
+
+NOTE: To disable collection of a specific resource type, simply exclude all metrics using the XX_metric_exclude. 
+For example, to disable collection of VMs, add this:
+```vm_metric_exclude = [ "*" ]```
+
+```
+# Read metrics from one or many vCenters
+[[inputs.vsphere]]
+  ## List of vCenter URLs to be monitored. These three lines must be uncommented
+  ## and edited for the plugin to work.
+  vcenters = [ "https://vcenter.local/sdk" ]
+  username = "user@corp.local"
+  password = "secret"
+
+  ############### VMs ###############
+  ## Typical VM metrics (if omitted or empty, all metrics are collected)
+  vm_metric_include = [
+    "cpu.demand.average",
+    "cpu.idle.summation",
+    "cpu.latency.average",
+    "cpu.readiness.average",
+    "cpu.ready.summation",
+    "cpu.run.summation",
+    "cpu.usagemhz.average",
+    "cpu.used.summation",
+    "cpu.wait.summation",
+    "mem.active.average",
+    "mem.granted.average",
+    "mem.latency.average",
+    "mem.swapin.average",
+    "mem.swapinRate.average",
+    "mem.swapout.average",
+    "mem.swapoutRate.average",
+    "mem.usage.average",
+    "mem.vmmemctl.average",
+    "net.bytesRx.average",
+    "net.bytesTx.average",
+    "net.droppedRx.summation",
+    "net.droppedTx.summation",
+    "net.usage.average",
+    "power.power.average",    
+    "virtualDisk.numberReadAveraged.average",
+    "virtualDisk.numberWriteAveraged.average",
+    "virtualDisk.read.average",
+    "virtualDisk.readOIO.latest",
+    "virtualDisk.throughput.usage.average",
+    "virtualDisk.totalReadLatency.average",
+    "virtualDisk.totalWriteLatency.average",
+    "virtualDisk.write.average",
+    "virtualDisk.writeOIO.latest",
+    "sys.uptime.latest",
+  ]
+  # vm_metric_exclude = [] ## Nothing is excluded by default
+  # vm_instances = true ## true by default
```

Comment:
We can remove the booleans `vm_instances = true`, `host_instances = true`, etc now and just use the include/exclude variables, right?

Reply:
No, see earlier comment! These flags control whether we collect what's known as instance metrics, e.g. metrics for specific CPU cores, NICs etc. This flag is important.